packages/std/src/results/subcall.rs

```diff
@@ -45,6 +47,69 @@ where
     pub reply_on: ReplyOn,
 }
 
+/// This is used for cases when we use ReplyOn::Never and the id doesn't matter
+pub const UNUSED_MSG_ID: u64 = 123456789;
+
+/// We implement thisas a shortcut so all existing code doesn't break.
+/// Up to 0.14, we could do something like:
+///   let messages = vec![BankMsg::Send { .. }.into()];
+/// In order to construct the response.
+///
+/// With 0.15, we move to requiring SubMsg there, but this allows the same
+/// `.into()` call to convert the BankMsg into a proper SubMsg with no reply.
+impl<M, T> From<M> for SubMsg<T>
+where
+    M: Into<CosmosMsg<T>>,
+    T: Clone + fmt::Debug + PartialEq + JsonSchema,
+{
+    #[inline]
+    fn from(msg: M) -> SubMsg<T> {
+        call(msg)
+    }
+}
+
+/// call takes eg. BankMsg::Send{} and wraps it into a SubMsg with normal message sematics (no reply)
+pub fn call<M, T>(msg: M) -> SubMsg<T>
```

Comment:
I started with static functions, then felt that was too long and awkward.
We also provide `coin()` not `Coin::create()` as a helper for a very common action, which is much nicer for app devs.
I think it is very reasonable to create a helper for this case.
I am however in agreement with moving `subcall()` -> `SubMsg::new()` and `subcall_with_limit()` -> `SubMsg::new_with_limit()`.

> Let's avoid the term "call" as this implies a synchonous function call.

Yes. I didn't have a better name that captured the function. Happy for more suggestions (and `new_never` is more confusing for people who have not been using submessages)

Reply:
> We also provide coin() not Coin::create() as a helper for a very common action, which is much nicer for app devs.

This is shorter, but I'm not sure it is nicer. It might be nicer for you because you know that it does and use it a lot. But each new public symbol needs to be tought and remembered by app developers. `Coin::create` is self-explaining, `SubMsg::new_*` as well.

Reply:
Some renames in https://github.com/CosmWasm/cosmwasm/pull/961/commits/748a4b5fe10a77a3412bd8b056a313511fe55a9d